.github/workflows/sanity.yaml

```diff
@@ -21,7 +21,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Run linting checks
+      - name: Run golangci linting checks
         uses: "golangci/golangci-lint-action@v2"
         with:
           version: "v1.45.2"
+
+      - name: Lint markdown files
```

Comment:
Should we move this into its own individual job.

Reply:
Sure, it probably makes more sense that way

Reply:
Actually, I kind of like having the markdown lint step be under the `lint` job since its technically also linting. The two lint steps share the same checkout action too which is nice. I don't know that we want a `sanity/markdown` job to run standalone, as it kind of clutters the UI, versus just keeping the markdown linter as part of the `sanity/lint` job and diagnosing any failures there as general lint errors.

Reply:
Yep that's fine with me. The only consideration here is that multiple steps within an individual job are run sequentially vs. jobs which run in parallel.

Reply:
Good point -- I think since the markdown job is fairly quick we can get away with this, but it's worth keeping in mind.